### PR TITLE
docs: TT-301 Update readme documentation and add Time Tracker CLI docs

### PR DESCRIPTION
**Pull request by jcalarcon98**
## Description

All the README documentation was updated to support the new way of running the project, in this case, everything was run with docker-compose, also documentation was added for the correct use of the "Time Tracker CLI", in charge of generating data fake in the cosmos emulator.

### CODE REVIEW

**Review by DrFreud1 — APPROVED** [review]
It seems very good for me. Everything seems to be pretty clear and understandable. Great Job @jcalarcon98 !

File: README.md [hunk 1]
```diff
@@ -10,135 +10,183 @@ Follow the following instructions to get the project ready to use ASAP.
...
 
-  In Unix based operative systems:
+```shell
+docker-compose up --build
+```
 
-  ```
-  #Create virtual enviroment
-  virtualenv .venv
+This command will build all images with the necessary configurations for each one, aslo
```
Comment (edgardavid2015) [inline README.md:59]: At the end of the line you wrote "aslo", I think the correct word is "also".

File: cosmosdb_emulator/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+![image](https://user-images.githubusercontent.com/56373098/127604274-041c2af7-d7a8-4b8d-b784-8280773b68c8.png)
+
+Where you can see the actions we can perform on a given Entity:
+
+Currently, the CLI only allows the creation of Time-entries and allows the deletion of any entity.
+
+Available Actions:
+
+- Create: Allows creating new fake data about a certain entity.
+- Delete: Allows creating cosmos data about a certain entity.
```
Comment (edgardavid2015) [inline cosmosdb_emulator/README.md:48]: In the Delete action you wrote: "Allows creating....". In this case it should say "Allows deleting....".

**Review by edgardavid2015 — COMMENTED** [review]
Excellent work Jean Carlos, I just saw two mistakes in the README files.

File: requirements/time_tracker_api/dev.txt [hunk 3]
```diff
@@ -14,9 +14,6 @@ pytest-mock==2.0.0
 # Coverage
 coverage==4.5.1
 
-# Git hooks
```
Comment (scastillo-jp) [inline requirements/time_tracker_api/dev.txt:17]: @jcalarcon98 why are you removing this lines?

File: README.md (hunk 1 above)
Comment (jcalarcon98) [reply README.md:59]: My mistake, I will correct it, thank you very much for checking.

**Review by JosueOb — APPROVED** [review]
I didn't have any complications following the instructions to set up the backend development environment using Docker. Also, I tested the time tracker CLI and was able to create new entries using the interactive and command-based way.

File: cosmosdb_emulator/README.md (hunk 2 above)
Comment (jcalarcon98) [reply cosmosdb_emulator/README.md]: Thank you Edgar, I overlooked it, solved 💯

File: requirements/time_tracker_api/dev.txt (hunk 3 above)
Comment (jcalarcon98) [reply requirements/time_tracker_api/dev.txt:17]: Explained in an internal conversation, solved 💯

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**